xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourneyFrequency_version.xsd

```diff
@@ -212,26 +212,52 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Elements for a JOURNEY FREQUENCY GROUP.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="FirstDepartureTime" type="xsd:time">
-				<xsd:annotation>
-					<xsd:documentation>Time of first departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="FirstDayOffset" type="DayOffsetType" default="0" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Offset days for end time. Number of days after the starting operational day if journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="LastDepartureTime" type="xsd:time" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Time of last departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="LastDayOffset" type="DayOffsetType" default="0" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Offset days for end time. Number of days after the starting departure time of the journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
+			<xsd:choice>
+				<xsd:sequence>
+					<xsd:element name="FirstDepartureTime" type="xsd:time">
+						<xsd:annotation>
+							<xsd:documentation>Time of first departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+					<xsd:element name="FirstDayOffset" type="DayOffsetType" default="0" minOccurs="0">
+						<xsd:annotation>
+							<xsd:documentation>Offset days for start time. Number of days after the starting operational day if journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+					<xsd:element name="LastDepartureTime" type="xsd:time" minOccurs="0">
+						<xsd:annotation>
+							<xsd:documentation>Time of last departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+					<xsd:element name="LastDayOffset" type="DayOffsetType" default="0" minOccurs="0">
+						<xsd:annotation>
+							<xsd:documentation>Offset days for end time. Number of days after the starting departure time of the journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
```

Comment:
```suggestion
							<xsd:documentation>Offset days for the last departure time. Number of days after the last departure time of the journey if not same calendar day. Default is 0 for the same day.</xsd:documentation>
```

Reply:
Hm, I'm confused anew - how can, for the" last departure time", the number of days after the "last departure time" be non-zero?

Reply:
Because the last departure time may be 25:00, which is tonight at 1am. For the operating day it will be December 13th, but it is _actually_ December 14th.